.github/workflows/PR_summary.yml

```
@@ -31,8 +31,12 @@ jobs:
           # we use curl rather than octokit/request-action so that the job won't fail
           # (and send an annoying email) if the labels don't exist
           curl --request POST \
-            --url https://api.github.com/repos/${{ github.repository }}/issues/${{ github.event.pull_request.number }}/labels/merge-conflict \
+            --header 'Accept: application/vnd.github+json' \
             --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}'
```

```suggestion
            --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' \
```
Maybe this will fix it??